compiler/parser/src/tokenizer/mod.rs

```diff
@@ -83,7 +83,6 @@ mod tests {
     test_ident
     12345
     address
-    async
```

Comment:
I see `async` was removed from this tokenizer test but it is still in lexer.rs on line 397, is that so that it can detect the deprecated feature and report on it?  If so maybe it should remain in the tokenizer test until taken out of the lexer?